tests/integration/components/layer-record-views/tax-lot/intersecting-layers-views-test.js

```diff
@@ -4,15 +4,20 @@ import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 import { setupMirage } from 'ember-cli-mirage/test-support';
 
-module('Integration | Component | layer-record-views/tax-lot/intersecting-layers-views', function(hooks) {
-  setupRenderingTest(hooks);
-  setupMirage(hooks);
+module(
+  'Integration | Component | layer-record-views/tax-lot/intersecting-layers-views',
+  function (hooks) {
+    setupRenderingTest(hooks);
+    setupMirage(hooks);
 
-  test('it shows none found with empty response', async function(assert) {
-    this.server.get('https://planninglabs.carto.com/api/v2/sql', () => ({ rows: [] }));
+    test('it shows none found with empty response', async function (assert) {
+      this.server.get('https://planninglabs.carto.com/api/v2/sql', () => ({
+        rows: [],
+      }));
 
-    await render(hbs`{{layer-record-views/tax-lot/intersecting-layers-views}}`);
+      await render(hbs`<LayerRecordViews::TaxLot::IntersectingLayersViews />`);
```

Comment:
@allthesignals Here is another test that I haven't been able to get back to passing. For context, I replaced the component from `ember-content-placeholders` with just the normal spinner icon in  `intersecting-layers.hbs` because that ember addon is abandoned. It seems to work fine with the loading spinner when I test it manually but for some reason this test doesn't seem to "wait" for the mocked async call before checking for `"None found"`. Could you take a crack at this one?